test/options/sourcecodefolderpath-azure-sources/Expected/AcceptanceTests/ParameterFlattening/package.json

```diff
@@ -17,9 +17,9 @@
   "homepage": "http://github.com/azure/azure-sdk-for-node",
   "repository": {
     "type": "git",
-    "url": "https://github.com/azure/azure-sdk-for-node.git"
+    "url": "http://github.com/azure/azure-sdk-for-node.git"
```

Comment:
not https?

Reply:
Done in https://github.com/Azure/autorest.nodejs/pull/55/commits/843ecab4e639669eaea5392d588aae18b83c58fd.